src/compiler/checker.ts

```diff
@@ -4963,6 +4963,10 @@ namespace ts {
             return checkTypeRelatedTo(source, target, identityRelation, /*errorNode*/ undefined) ? Ternary.True : Ternary.False;
         }
 
+        function compareTypesAssignable(source: Type, target: Type): Ternary {

```

Comment:
I'm not sure about this name... what is it trying to say that is different from isTypeAssignableTo?